### PR TITLE
update for iris

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: governance-aepp/src/utils/aeternity.js [hunk 1]
```diff
@@ -147,7 +154,7 @@ aeternity.initWalletSearch = async (successCallback) => {
     name: 'AEPP',
     nodes: [
       {name: 'ae_mainnet', instance: await Node({url: settings.ae_mainnet.nodeUrl})},
-      //{name: 'ae_uat', instance: await Node({url: settings.ae_uat.nodeUrl})}
+      {name: 'ae_uat', instance: await Node({url: settings.ae_uat.nodeUrl})}
```
Comment (kenodressel) [inline governance-aepp/src/utils/aeternity.js:157]: should we leave this active?

File: governance-aepp/src/assets/contracts/Poll_Iris.aes [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  entrypoint has_voted(voter : address) : bool = Map.member(voter, state.votes)
+
+  entrypoint is_closed() : bool =
+    switch(state.close_height)
+      Some(close_height) => Chain.block_height >= close_height
+      None => false
+
+  entrypoint voted_option(voter : address) : option(int) = Map.lookup(voter, state.votes)
+
+  entrypoint version() : int = 1
```
Comment (kenodressel) [inline governance-aepp/src/assets/contracts/Poll_Iris.aes:94]: shouldn't we bump the version?

File: governance-aepp/src/utils/aeternity.js (hunk 1 above)
Comment (thepiwo) [reply governance-aepp/src/utils/aeternity.js:157]: good question, let me check

File: governance-aepp/src/assets/contracts/Poll_Iris.aes (hunk 2 above)
Comment (thepiwo) [reply governance-aepp/src/assets/contracts/Poll_Iris.aes:94]: :+1:

File: governance-aepp/src/utils/aeternity.js (hunk 1 above)
Comment (thepiwo) [reply governance-aepp/src/utils/aeternity.js:157]: it should be included, just removed it earlier for compatibility, but seems to be a non-issue now

**Review by kenodressel — APPROVED** [review]

**Pull request merged**